### PR TITLE
Added a log for zipper.

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: src/MessageHandler/ZipDatasetFilesHandler.php [hunk 1]
```diff
@@ -97,13 +97,16 @@ public function __invoke(ZipDatasetFiles $zipDatasetFiles)
         $datasetSubmission = $this->entityManager->getRepository(DatasetSubmission::class)->find($datasetSubmissionId);
         $filesInfo = $this->fileRepository->getFilePathNameAndPhysicalPath($fileIds);
         $destinationPath = $this->downloadDirectory . DIRECTORY_SEPARATOR .  str_replace(':', '.', $datasetSubmission->getDataset()->getUdi()) . '.zip';
+        $this->logger->info('Created: ' . $destinationPath);
```
Comment (praneethpr) [inline src/MessageHandler/ZipDatasetFilesHandler.php:100]: Maybe change the wording to something like "Zipping started"

File: src/MessageHandler/ZipDatasetFilesHandler.php [hunk 2]
```diff
@@ -97,13 +97,16 @@ public function __invoke(ZipDatasetFiles $zipDatasetFiles)
...
         try {
             $outputStream = array('fileStream' => fopen($destinationPath, 'w+'));
             $this->zipFiles->start($outputStream, basename($destinationPath));
             foreach ($filesInfo as $fileItemInfo) {
+                $this->logger->info("adding file to $destinationPath:" . $fileItemInfo['filePathName']);
                 $this->zipFiles->addFile($fileItemInfo['filePathName'], $this->datastore->getFile($fileItemInfo['physicalFilePath']));
             }
             $this->zipFiles->finish();
+            $this->logger->info('Closed: ' . $destinationPath);
```
Comment (praneethpr) [inline src/MessageHandler/ZipDatasetFilesHandler.php:109]: Same here

File: src/MessageHandler/ZipDatasetFilesHandler.php [hunk 3]
```diff
@@ -91,27 +91,27 @@ public function __construct(
      */
     public function __invoke(ZipDatasetFiles $zipDatasetFiles)
     {
-        $this->logger->info('Zipping files started');
```
Comment (mickel1138) [inline src/MessageHandler/ZipDatasetFilesHandler.php:94]: Why was this moved? The process started. Maybe add this logger back, but say something like "zip worker started" or something.

File: src/MessageHandler/ZipDatasetFilesHandler.php [hunk 4]
```diff
@@ -91,27 +91,27 @@ public function __construct(
...
+            $this->logger->info('Zipping ended: ' . $destinationPath);
             fclose($outputStream['fileStream']);
             $datasetSubmission->getFileset()->setZipFilePath($destinationPath);
             $this->entityManager->flush();
         } catch (\Exception $exception) {
             $this->logger->error(sprintf('Unable to zip file. Message: %s', $exception->getMessage()));
             return;
         }
-
-        $this->logger->info(sprintf('Zipping files done'));
```
Comment (mickel1138) [inline src/MessageHandler/ZipDatasetFilesHandler.php:115]: Similar to comment above. You can say that the worker is done.

**Review by praneethpr — APPROVED** [review]

File: src/MessageHandler/ZipDatasetFilesHandler.php (hunk 4 above)
Comment (mickel1138) [reply src/MessageHandler/ZipDatasetFilesHandler.php:115]: It seems that me a Praneeth were saying the same thing. But I feel the start worker messages should be the first line of code. What if things break before them.

**Review by mickel1138 — APPROVED** [review]
Tested and works.

**Review by mickel1138 — APPROVED** [review]

**Review by praneethpr — APPROVED** [review]
